packages/sampler/stories/qa-stories/Marquee.js

```diff
@@ -39,6 +39,7 @@ storiesOf('Marquee', module)
 					<Marquee
 						style={{width: ri.unit(399, 'rem')}}
 						disabled={disabled}
+						forceDirection={select('forceDirection', ['', 'ltr', 'rtl'], Marquee, '')}
```

Comment:
Do we need `nullify()` on these?

Reply:
It's handled in `enact-knobs/select.js`
https://github.com/enactjs/enact/blob/develop/packages/sampler/src/enact-knobs/select.js#L54

Reply:
:awesome: